test/unit/controllers/declaration/OfficeOfExitControllerSpec.scala

```diff
@@ -230,15 +231,15 @@ class OfficeOfExitControllerSpec extends ControllerSpec with OptionValues {
 
         withNewCaching(request.cacheModel)
 
-        val officeOfExitInput = Some("GB123456");
+        val officeOfExitInput = Some("GB123456")
         val isUKOfficeOfExitInput = AllowedUKOfficeOfExitAnswers.yes
 
         val correctForm = Json.toJson(OfficeOfExitInsideUK(officeOfExitInput, isUKOfficeOfExitInput))
 
         val result = controller.saveOffice(Mode.Normal)(postRequest(correctForm))
 
         await(result) mustBe aRedirectToTheNextPage
-        thePageNavigatedTo mustBe controllers.declaration.routes.PreviousDocumentsController.displayPage()
+        thePageNavigatedTo mustBe controllers.declaration.routes.PreviousDocumentsSummaryController.displayPage()
```

Comment:
Similar here. Should it not be the first page for Documents?

Reply:
Similar like above